python/ray/_private/worker.py

```diff
@@ -3275,7 +3275,7 @@ def method(self):
             By default it is empty.
         accelerator_type: If specified, requires that the task or actor run
             on a node with the specified type of accelerator.
-            See `ray.util.accelerators` for accelerator types.
+            See `ray.util.accelerators <https://github.com/ray-project/ray/blob/master/python/ray/util/accelerators/accelerators.py>`_ for accelerator types. # noqa: E501
```

Comment:
Can you move the comment to the very end of the docstring? I _think_ this should capture it for the whole docstring and not have it render inline.

```
"""  # noqa: E501
```